### PR TITLE
Added a section in docs about decorators

### PR DESCRIPTION
**Pull request by sandeshbhusal**
JWT and authorization can be a tough topic for beginners to break into. While the documentation clearly outlines the starting places, it has left out the portion for decorators and decorating the to-be-protected views.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -97,6 +97,21 @@ Now in order to access protected api urls you must include the `Authorization: B
 $ curl -H "Authorization: Bearer <your_token>" http://localhost:8000/protected-url/
 ```
 
+In addition to adding the Authorization: Bearer in your requests, make sure that you have decorated your views properly. For this, import the JSONWebTokenAuthentication authenticaion class from rest_framework_jwt.authentication
```
Comment (fitodic) [inline docs/index.md:100]: > JSONWebTokenAuthentication authenticaion

"authenticaion" is misspelled, and the JSONWebTokenAuthentication can be styled as `JSONWebTokenAuthentication`

> make sure that you have decorated your views properly

I suggest you point out that this only applies to function based views. The class based views include this by overriding the `authentication_classes` class attribute. Feel free to add an example for the class based views as well.

**Comment by fitodic** [comment]
Closes https://github.com/Styria-Digital/django-rest-framework-jwt/issues/66

File: docs/index.md (hunk 1 above)
Comment (sandeshbhusal) [reply docs/index.md:100]: Sure! I will modify the PR according to your suggestions!

**Comment by sandeshbhusal** [comment]
I just made the changes. Could you review the changes? Thanks!

File: docs/index.md [hunk 2]
```diff
@@ -97,6 +97,28 @@ Now in order to access protected api urls you must include the `Authorization: B
 $ curl -H "Authorization: Bearer <your_token>" http://localhost:8000/protected-url/
 ```
 
+In addition to adding the Authorization: Bearer in your requests, make sure that you have decorated your views properly. For this, import the `JSONWebTokenAuthentication` authentication class from `rest_framework_jwt.authentication`
```
Comment (fitodic) [inline docs/index.md:100]: You could also point out here that this is necessary if the `JSONWebTokenAuthentication` class wasn't added to the `DEFAULT_AUTHENTICATION_CLASSES`, or if JWT authentication is needed only on certain views.

Comment (sandeshbhusal) [reply docs/index.md:100]: I just added a changelog file. Please review! 

(Also this is my first contribution on github. Thank you for being so helpful!)

Comment (fitodic) [reply docs/index.md:100]: Congrats for your first contribution :slightly_smiling_face: 
As to the
> You could also point out here that this is necessary if the JSONWebTokenAuthentication class wasn't added to the DEFAULT_AUTHENTICATION_CLASSES, or if JWT authentication is needed only on certain views.

I don't see it anywhere. Did you forget to push that commit?

Comment (sandeshbhusal) [reply docs/index.md:100]: I am really sorry, because of my unstable internet I have not been able to complete this. I will do this ASAP!

Comment (sandeshbhusal) [reply docs/index.md:100]: I have added another commit that has addressed this missing part in the doc. Can you take a look?

Comment (fitodic) [reply docs/index.md:100]: Great, thanks!

**Pull request merged**